lib/minesweeper_2pl/cli.rb

```diff
@@ -1,4 +1,178 @@
-module Minesweeper_2pl
+module Minesweeper
   class CLI
+    def print(msg)
+      puts msg
+    end
+
+    def start
+      welcome
+      get_player_params
+    end
+
+    def get_move(game)
+      ask_for_move
+      get_player_input(game)
+    end
+
+    def get_player_params
+      result = []
+      size = nil
+      while size.nil?
+        ask_for_board_size
+        size = get_player_entered_board_size
+      end
+      result << size
+      count = nil
+      while count.nil?
+        ask_for_bomb_count(size)
+        count = get_player_entered_bomb_count(size * size)
+      end
+      result << count
```

Comment:
These two blocks of code look functionally identical:

- set a variable to `nil`
- get some input from the user and store it in the variable
- when you get some input, store it in `result`

Is there a way to extract this logic to a named helper method?